src/styles/index.scss

```diff
@@ -1,3 +1,143 @@
-body {
+  @import'/src/styles/stars';
+
+* {
+   box-sizing: border-box;
+ }
+
+ body,
+ h2,
+ p,
+ div{
   margin: 0;
-}
+ }
+
+ .card {
+   width: 200px;
+  //  height: 406px;
+   flex-shrink: 0;
+   border-radius: 5px;
+   border: 1px solid #f3f3f3;
+   background: #fff;
+   position: relative;
+
+   &__image {
+     margin-top: 32px;
+     margin-left: 19px;
+     width: 160px;
+     height: 134px;
+     background-image: url(../images/imac.jpeg);
+     background-size: cover;
+     background-position: center;
```

Comment:
Not fixed
```suggestion

```
![2024-01-03_15h19_58](https://github.com/mate-academy/layout_product-cards/assets/61904995/112d9e7a-99f9-4895-b3a6-d16696ff7f4c)